### ISSUE TITLE
Python Extension Testing Error

### ISSUE THREAD

**Issue by j40903272** [trendmicro/tlsh#69]
When I run the command ./python_test.sh, it gives following error :
```testing the -force option for short strings...

python ../py_ext/tlsh_digest.py -force example_data/small.txt > tmp/py_small.tls                                                                                     h
diff tmp/py_small.tlsh exp/small.tlsh_EXP
diff: exp/small.tlsh_EXP: No such file or directory
error: diff tmp/py_small.tlsh exp/small.tlsh_EXP
```

It seems that the testing validation file in exp/ is missing.

**Comment by StanPlatinum** [comment]
Has anyone known how to fix it?

Thanks!

**Comment by jonjoliver** [comment]
fixed in version 4.5
